static/js/tippyjs.js

```diff
@@ -32,7 +32,7 @@ tippy.setDefaultProps({
 
     // This has the side effect of some properties of parent applying to
     // tooltips.
-    appendTo: "parent",
+    appendTo: document.body,
 
```

Comment:
This is a bad change. Why do you think  you need it here?

Reply:
Changing it makes all tooltips append/show over any popover that is open. It was done because the `Add Stream` menu was appending over the `Filter Stream` tooltip but it could be done the opposite way too it that sounds good?

Reply:
Let's remove this change. There is a strong reason I have used "parent" here - to avoid tooltips from persisting after the element is hidden. I still don't get it why you made this change.

Reply:
The last line of the issue stated "The nearby "Filter streams" tooltip should appear on top of the menu.", this change was done to implement that. The other way that seems correct to me, to implement this is to update the `appendTo` property of `default_popover_props`. I have updated the PR @amanagr, requesting to review the PR.